specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/cosmos-db.json

```diff
@@ -0,0 +1,7956 @@
+          "$ref": "#/definitions/ConflictResolutionPolicy",
+          "description": "The conflict resolution policy for the graph."
+        }
+      },
+      "required": [
+        "id"
+      ]
+    },
+    "CreateUpdateOptions": {
+      "type": "object",
+      "properties": {
+        "throughput": {
+          "type": "integer",
+          "description": "Request Units per second. For example, \"throughput\": 10000."
+        },
+        "autoscaleSettings": {
+          "$ref": "#/definitions/AutoscaleSettings",
+          "description": "Specifies the Autoscale settings."
+        }
+      },
+      "description": "CreateUpdateOptions are a list of key-value pairs that describe the resource. Supported keys are \"If-Match\", \"If-None-Match\", \"Session-Token\" and \"Throughput\""
+    },
+    "AutoscaleSettings": {
+      "type": "object",
+      "properties": {
+        "maxThroughput": {
+          "type": "integer",
+          "description": "Represents maximum throughput, the resource can scale up to."
+        }
+      }
+    },
+    "Capability": {
+      "type": "object",
+      "description": "Cosmos DB capability object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the Cosmos DB capability. For example, \"name\": \"EnableCassandra\". Current values also include \"EnableTable\" and \"EnableGremlin\"."
+        }
+      }
+    },
+    "Tags": {
+      "type": "object",
+      "additionalProperties": {
+        "type": "string"
+      },
+      "description": "Tags are a list of key-value pairs that describe the resource. These tags can be used in viewing and grouping this resource (across resource groups). A maximum of 15 tags can be provided for a resource. Each tag must have a key no greater than 128 characters and value no greater than 256 characters. For example, the default experience for a template type is set with \"defaultExperience\": \"Cassandra\". Current \"defaultExperience\" values also include \"Table\", \"Graph\", \"DocumentDB\", and \"MongoDB\"."
+    },
+    "ManagedServiceIdentity": {
+      "properties": {
+        "principalId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The principal id of the system assigned identity. This property will only be provided for a system assigned identity."
+        },
+        "tenantId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The tenant id of the system assigned identity. This property will only be provided for a system assigned identity."
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of identity used for the resource. The type 'SystemAssigned, UserAssigned' includes both an implicitly created identity and a set of user assigned identities. The type 'None' will remove any identities from the virtual machine.",
+          "enum": [
+            "SystemAssigned",
```

Comment:
you don't support both system and user assigned?  The description indicates you do

Reply:
Hi, The "UserAssigned" is also in the enum below:
          "enum": [
            "SystemAssigned",
            "UserAssigned",
            "None"
          ],
So I believe I don't need any change here?

Reply:
Hi, I think what you mean is that the description is not accurate, and I have fixed it to be more concise.

Reply:
This enum will let people choose 1 (and only 1) of the values specified in the "enum" array.  They will not be able to select "SystemAssigned, UserAssigned" as a value which is needed to enable both on the same resource.  Search the repo for "userassigned" to see how others are representing these options in the enum

Reply:
Hi, thanks for the hint! It's done!